data/wanderers.txt

```diff
@@ -717,6 +737,9 @@ event "wanderers: tempest mass production"
 		remove variant
 			"Autumn Leaf"
 			"Summer Leaf"
+	fleet "Wanderer Miners"
+		add variant
+			"Tempest"
```

Comment:
As a low-weight variant (1 in 21 chance to be picked when the `Wanderer Miners` fleet is placed), it was my perception that this would appear as though one of the many Tempests in the system (from the `Wanderer Defense` fleets) was keeping itself busy while patrolling the system. The game's current mechanics will - in the absence of combat targets - rapidly instruct ships to either jump to a new system or land on a planet in the system. This generates traffic volume, but not necessarily a persistent presence (at least until combat starts).

Ideally there would not be any of the dedicated warship designs in the mining fleet, but the Wanderer's stated mining drone has no weapon space and there are no candidate weapons for it to use (e.g. a variant with a baked-in, low-power weapon, like how the Arfecta has a baked-in cloaking device). If someone can make (I don't do artwork well 😁) a lore-compatible mining beam for the Earth Shaper, I would happily rework the `Wanderer Miners` fleet to use it, and feature a rare Summer Leaf, with no Autumn Leaf / Tempests.